nodes/usb_cam_node.cpp

```diff
@@ -245,13 +251,31 @@ class UsbCamNode
     }
 
     // check auto exposure
-    if (!autoexposure_)
+    if (auto_balance_exposure_)
     {
-      // turn down exposure control (from max of 3)
-      cam_.set_v4l_parameter("exposure_auto", 1);
-      // change the exposure level
+      // changing auto exposure parameters 
+      cam_.set_v4l_parameter("exposure_auto", AUTO_EXPOSURE_AUTO_MODE);
+      std::this_thread::sleep_for(
+        std::chrono::seconds{ WAIT_CHANGING_AUTO_EXPOSURE }
+      );
+      cam_.set_v4l_parameter("exposure_auto", AUTO_EXPOSURE_MANUAL_MODE);
```

Comment:
Approved, but can you add a comment for the difference between this `auto_balance_exposure_` case and `!autoexposure_` below?

Reply:
changed, thanks